### PR TITLE
[Fix](nereids) report an error when a function that cannot be used for window analysis appears in the over clause

### PR DESCRIPTION
**Pull request by feiniaofeiafei**

### CODE REVIEW

**Comment by feiniaofeiafei** [comment]
run buildall

**Comment by feiniaofeiafei** [comment]
run buildall

File: fe/fe-core/src/main/java/org/apache/doris/nereids/rules/rewrite/ExtractAndNormalizeWindowExpression.java [hunk 1]
```diff
@@ -64,6 +66,10 @@ private Plan normalize(LogicalProject<Plan> project) {
                     if (output instanceof WindowExpression) {
                         WindowExpression windowExpression = (WindowExpression) output;
                         Expression expression = ((WindowExpression) output).getFunction();
+                        if (!(expression instanceof SupportWindowAnalytic)) {
+                            throw new AnalysisException("1OVER clause requires aggregate or analytic function: "
```
Comment (morrySnow) [inline fe/fe-core/src/main/java/org/apache/doris/nereids/rules/rewrite/ExtractAndNormalizeWindowExpression.java:70]:
```suggestion
                            throw new AnalysisException("OVER clause requires aggregate or analytic function: "
```

**Comment by feiniaofeiafei** [comment]
run buildall

**Comment by feiniaofeiafei** [comment]
run buildall

**Comment by feiniaofeiafei** [comment]
run buildall

**Review by morrySnow — APPROVED** [review]

**Pull request closed without merging**